DemoQA/cypress/pages/regisform/confirmForm.js

```diff
@@ -0,0 +1,68 @@
+export  const confirmForm ={
+    TXT_NAME_CF: "tbody tr:nth-child(1) td:nth-child(2)",
```

Comment:
You can try `TXT_VALUE: "td + td"` then `cy.get(this.TXT_VALUE).contains(value).should("be.visible")`. One locator can be used to implement all verify because you know how the result should be